### ISSUE TITLE
Commands from palette return “markJumpTo* not found”

### ISSUE THREAD

**Issue by Fred-Vatin** [spywhere/vscode-mark-jump#23]
VS Code version | OS | Extension version
------------------ | ---- | ----
1.37.1 | Win 10 x64 pro (french) 18362 | 0.8.2

It seems the commands in **_package.json_** are outdated. The red ones are shown in palette and return errors. The green ones works when assigning a custom keybindings. The other features works: show in explorer viewlet/status bar.
![screenshot](https://i.snipboard.io/Gw1ekA.jpg)

Edit the **_package.json_** file fixes the issue.
```javascript
		"commands": [
			{
				"command": "markJump.jumpToProjectMarks",
				"title": "Mark Jump: Jump to Project-wide Marks..."
			},
			{
				"command": "markJump.jumpToMarks",
				"title": "Mark Jump: Jump to Marks..."
			},
			{
				"command": "markJump.jumpToMarks.section",
				"title": "Mark Jump: Jump to Sections..."
			},
			{
				"command": "markJump.jumpToMarks.todo",
				"title": "Mark Jump: Jump to TODOs..."
			},
			{
				"command": "markJump.jumpToMarks.note",
				"title": "Mark Jump: Jump to Notes..."
			}
		],
		"keybindings": [
			{
				"command": "markJump.jumpToProjectMarks"
			},
			{
				"command": "markJump.jumpToMarks"
			},
			{
				"command": "markJump.jumpToEditorMarks.section",
				"when": "editorTextFocus"
			},
			{
				"command": "markJump.jumpToEditorMarks.todo",
				"when": "editorTextFocus"
			},
			{
				"command": "markJump.jumpToEditorMarks.note",
				"when": "editorTextFocus"
			},
			{
				"command": "markJump.jumpToPreviousMark",
				"when": "editorTextFocus"
			},
			{
				"command": "markJump.jumpToNextMark",
				"when": "editorTextFocus"
			}
		],
```

**Comment by spywhere** [comment]
Thanks for reporting this, I think it's quite a long time since I update `package.json` as I, most of the time, using this through the status bar.

**Comment by spywhere** [comment]
Fix in v0.9.0
